test/bvt.py

```diff
@@ -1,23 +1,22 @@
-import unittest
-import io
 import os
+import unittest
+
 import xml.etree.ElementTree as ET
 
 from tableaudocumentapi import Workbook, Datasource, Connection, ConnectionParser
 
-# Disable the 120 line limit because of the embedded XML on these lines
-# TODO: Move the XML into external files and load them when needed
 
-TABLEAU_93_WORKBOOK = '''<?xml version='1.0' encoding='utf-8' ?><workbook source-build='9.3.1 (9300.16.0510.0100)' source-platform='mac' version='9.3' xmlns:user='http://www.tableausoftware.com/xml/user'><datasources><datasource caption='xy (TestV1)' inline='true' name='sqlserver.17u3bqc16tjtxn14e2hxh19tyvpo' version='9.3'><connection authentication='sspi' class='sqlserver' dbname='TestV1' odbc-native-protocol='yes' one-time-sql='' server='mssql2012.test.tsi.lan' username=''></connection></datasource></datasources></workbook>'''  # noqa
+TABLEAU_93_TWB = 'test/assets/TABLEAU_93_TWB.twb'
```

Comment:
Yeah, that's less fragile.

It looks like it would be:

`os.path.join(
    os.path.dirname(__file__), 
    'asserts', 
    filename)
)`

No 'tests' directory, since bvt.py is already located in test.

`Since os.path.dirname(__file__)` is duplicated, how about:

`TEST_DIR = os.path.dirname(__file__)

TABLEAU_93_TWB = os.path.join(TEST_DIR, 'assets', 'TABLEAU_93_TWB.twb`
... etc